idd/Energy+.idd.in

```diff
@@ -6412,7 +6412,7 @@ WindowMaterial:Glazing:EquivalentLayer,
        \required-field
        \type choice
        \key SpectralAverage
-       \key Spectral (not supported now)
+       \note Spectral is no longer supported and SpectralAverage is now the default.
```

Comment:
@rglazer2 Did SpectralAverage actually get the \default tag here?  It doesn't show up, but it might be due to Github's diff viewer with the idd.